xwiki-platform-core/xwiki-platform-repository/xwiki-platform-repository-test/xwiki-platform-repository-test-utils/src/main/java/org/xwiki/repository/test/RepositoryTestUtils.java

```diff
@@ -319,4 +317,10 @@ public void waitUntilReady() throws Exception
         // Make sure Solr queue is empty
         this.solrUtils.waitEmpyQueue();
     }
+
+    public ObjectEditPage gotoExtensionObjectsEditPage(String extensionName) {
+        LocalDocumentReference extensionPageReference = getExtensionPageReference(extensionName);
+        testUtils.gotoPage(extensionPageReference, "edit", ImmutableMap.of("editor","object"));
```

Comment:
No need to use Guava. Collections.singletonMap("editor", "object") should have the same effect.